mteb/tasks/Classification/eng/LegalBenchClassification.py

```diff
@@ -0,0 +1,1056 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class CanadaTaxCourtOutcomesLegalBenchClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="CanadaTaxCourtOutcomesLegalBenchClassification",
+        description="Canada Tax Court Outcomes LegalBench Classification Dataset",
+        reference="https://huggingface.co/datasets/nguha/legalbench",
+        dataset={
+            "path": "nguha/legalbench",
+            "name": "canada_tax_court_outcomes",
+            "revision": "12ca3b695563788fead87a982ad1a068284413f4",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="accuracy",
+        date=("2023-08-23", "2023-08-23"),
+        form=["written"],
+        domains=["Legal"],
+        task_subtypes=[],
+        license="cc-by-nc-4.0",
+        socioeconomic_status="high",
+        annotations_creators="expert-annotated",
+        dialect=["en-CA"],
+        text_creation="found",
+        bibtex_citation="""
+        @misc{guha2023legalbench,
+            title={LegalBench: A Collaboratively Built Benchmark for Measuring Legal Reasoning in Large Language Models}, 
+            author={Neel Guha and Julian Nyarko and Daniel E. Ho and Christopher Ré and Adam Chilton and Aditya Narayana and Alex Chohlas-Wood and Austin Peters and Brandon Waldon and Daniel N. Rockmore and Diego Zambrano and Dmitry Talisman and Enam Hoque and Faiz Surani and Frank Fagan and Galit Sarfaty and Gregory M. Dickinson and Haggai Porat and Jason Hegland and Jessica Wu and Joe Nudell and Joel Niklaus and John Nay and Jonathan H. Choi and Kevin Tobia and Margaret Hagan and Megan Ma and Michael Livermore and Nikon Rasumov-Rahe and Nils Holzenberger and Noam Kolt and Peter Henderson and Sean Rehaag and Sharad Goel and Shang Gao and Spencer Williams and Sunny Gandhi and Tom Zur and Varun Iyer and Zehua Li},
+            year={2023},
+            eprint={2308.11462},
+            archivePrefix={arXiv},
+            primaryClass={cs.CL}
+            }""",
+        n_samples={"test": 244},
+        avg_character_length={"test": 622.60},
+    )
+
+    def dataset_transform(self):
+        self.dataset = self.dataset.rename_column("answer", "label")
+
+
+class ContractNLIConfidentialityOfAgreementLegalBenchClassification(
```

Comment:
Some `ContractNLI*LegalBenchClassification`  results are missing

Reply:
Only the results for [Contract NLI Sharing with third-parties](https://hazyresearch.stanford.edu/legalbench/tasks/contract_nli_sharing_with_third-parties.html) [Contract NLI Survival of Obligations](https://hazyresearch.stanford.edu/legalbench/tasks/contract_nli_survival_of_obligations.html) have been included here. The others have been already added in #515 and #571.